.sanity-config.ini

```diff
@@ -196,7 +196,8 @@ disable=print-statement,
         unused-argument,
         too-few-public-methods,
         too-many-return-statements,
-        too-many-branches
+        too-many-branches,
+        duplicate-code
```

Comment:
I don't like disabling these two. They're annoying, but are good indicator that something is unmaintainable. Did a pylint upgrade increase the strictness or is there a specific block that is giving you trouble?

Reply:
Yes `pylint` is definitely more strict than a few versions back. And I'm not talking about one duplicated code block but like 50 or more and also imports included (look here for more info: https://app.circleci.com/pipelines/github/xlab-si/xopera-opera/1090/workflows/e1a35c29-d220-403a-a861-319ed75c7a75/jobs/2472) which would mean a lot of effort to reconstruct the code as it also detects duplicates on places that are not duplicates technically. So I tried to mark duplicated blocks with "#pylint: disable: duplicate-code" but it's just not working for me, so I had no other way than disabling these duplicates globally (I have also found this issue: https://github.com/PyCQA/pylint/issues/214 that seems to be connected to our problems).

Reply:
Aha and by the way I didn't disable `too-many-branches` but just `duplicate-code`, maybe a comma after the `duplicate-code` has confused you.

Reply:
I've definitely gotten confused by the comma :) I see green, I see new.

Looking at this a bit more, let's keep duplicate-code disabled. It doesn't add anything here because it marks import blocks as duplicates, and that's just insane.